lib/charms/pgbouncer_operator/v0/pgb.py

```diff
@@ -27,19 +27,26 @@
 from collections.abc import MutableMapping
 from configparser import ConfigParser, ParsingError
 from copy import deepcopy
-from typing import Dict
+from typing import Dict, Union
 
 logger = logging.getLogger(__name__)
 
 PGB_DIR = "/var/lib/postgresql/pgbouncer"
+PORT_MAX = 49151  # Maximum valid port number before we get into ephemeral ports
```

Comment:
is this variable being used?

Reply:
This was going to be a part of how we set `listen_port` in the config, but it's no longer relevant.